backend/src/libs/packages/config/config.package.ts

```diff
@@ -0,0 +1,77 @@
+import { type Config as TConfig } from 'convict';
+import convict from 'convict';
```

Comment:
Why not to import it like 
```suggestion
import convict, { type Config as TConfig } from 'convict';
```